MapboxCoreNavigation.podspec

```diff
@@ -42,7 +42,7 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxNavigationNative", "~> 3.1"
   s.dependency "MapboxDirections.swift", "~> 0.24.0"
```

Comment:
Ideally, we should be dropping the patch version ... it could cause issues down the road. Please update to `0.24`

Reply:
`0.24` resolves to `0.99.99`. `0.24.0` resolves to `0.24.99`.

Reply:
Ok. got it!

Reply:
Indeed - http://guides.cocoapods.org/using/the-podfile.html